UI/setup/select_coa.html

```diff
@@ -24,11 +24,19 @@ <h2><?lsmb text('Database Management Console') ?></h2>
            label = text('Chart of accounts')
     };
 ELSE;
+    lang = '';
+    IF languages;
+        lang = languages.pop;
+        IF lang.search('-');
```

Comment:
What's this code doing?? What problem is this solving? 'en-US' and 'en' are both in our language set, no? If not, why not? And why would this then be the best solution to that problem?